bench_test.go

```diff
@@ -67,11 +67,13 @@ func benchUnary(b *testing.B, handleropts []connect.HandlerOption, clientopts []
 func benchStreaming(b *testing.B, handleropts []connect.HandlerOption, clientopts []connect.ClientOption) {
 	b.Helper()
 	_, client := startBenchServer(handleropts, clientopts)
-	req := &pingv1.CumSumRequest{Number: 12}
+	req := &pingv1.PingStreamRequest{
+		Data: []byte("Hello, otel!"),
+	}
 	b.ResetTimer()
 	b.RunParallel(func(pb *testing.PB) {
 		for pb.Next() {
-			stream := client.CumSum(context.Background())
+			stream := client.PingStream(context.Background())
```

Comment:
Hm. I didn't flag this earlier, but shouldn't we should be closing these clients?